src/Intl/Grapheme/Grapheme.php

```diff
@@ -181,7 +189,7 @@ public static function grapheme_strstr($s, $needle, $beforeNeedle = false)
     private static function grapheme_position($s, $needle, $offset, $mode)
     {
         $needle = (string) $needle;
-        if (!preg_match('/./us', $needle)) {
+        if (80000 > \PHP_VERSION_ID && !preg_match('/./us', $needle)) {
```

Comment:
what about the similar regexp 4 lines later on the string ?

And what happens in PHP 8 for non-unicode strings ?

Reply:
The corresponding change in PHP was that an empty needle is now allowed which caused this check to fail. Now that you bring it up, I think I should have solved this differently. I'll submit a PR in the evening.